communications/meetings.md

```diff
@@ -0,0 +1,56 @@
+# Meetings
+
+Use the guides below as a starting point, and adapt for specific situations accordingly.
+
+## Before A Meeting
+
+Objectives:
+- Give all participants as much context as possible before the meeting.
+- _Asynchronously_ resolve as many items as possible before the meeting.
+- Determine, and possibly _reduce_, the scope, and length of the meeting.
+- Determine, and possibly _eliminate_, reasons to participate in the meeting.
+- Prepare the meeting set up to be as convenient as possible for all participants.
+- Crowdsource bookkeeping effort to increase accuracy, and minimize bias.
+
+Guide:
+- Preferably 1 work-day before the meeting, initiate one discussion per agenda item.
+    - The discussion can be a Slack thread, GitHub issue, Asana task, Google Docs, or anything with an interactive "comments" component.
+    - For a recurring meeting with past sessions, use the ones from the previous meeting session as a starting point. Remove ones no longer needed.
+    - For a net-new meeting, the organizer(s) should define the initial scope of items.
+    - Optionally initiate an extra, open-ended discussion to collect additional topics from all participants, graduate solidified ones to have their discussions.
+- In each discussion, participants write down briefing, information, knowledge, and ideas that are relevant.
+    - For a recurring meeting with past sessions, focus on briefings of pending items from the previous meeting session, such as what each _has done_, _has been doing_, and _plans to do_. Give everyone else a sense of _where you are_ and _the direction you're heading to_. This helps others to determine if there are potential directional conflicts/blocks.
+    - Raise relevant questions, problems, obstacles, unknowns, and any other issues. Try to make them demonstratable, by leveraging modern tools such as screen-captured images or videos, and/or steps to reproduce these issues.
+    - Preemptively discuss. Seek to resolve as many issues and directional conflicts/blocks as possible.
+- Cross off resolved (or portions of) items from the list of the meeting agenda. In the event of having all the item discussions resolved, skip to [section: After A Meeting](#after-a-meeting).
+- Put a tentative timebox on each unresolved item. This can be an outcome of a collective agreement.
+- Send a calendar invite to all participants, with Zoom/Hangouts Meet.
+
+## During A Meeting
+
+Objectives:
+- Maximize the gain of synchronous/in-person/voice communication to resolve items.
```

Comment:
Need to define what is the actual gain for in-person communication. Suggestion for this part:
```
* Align & inform all participants of relevant information at the same time (synchronously)
* Maintain time for each agenda item & scope of meeting as intended
* Record as many details as possible for future context/meetings
* Crowdsource bookkeeping effort to increase accuracy, and minimize bias.
```